_posts/2021-09-02-gsoc-final-report-estefania.md

```diff
@@ -0,0 +1,299 @@
+---
+layout: post
+title: "GSoC Report: Curvature for MDA Universes"
+---
+
+The main goal of my [GSoC project](https://summerofcode.withgoogle.com/projects/#5098282306502656)
+was to develop a tool to calculate membrane curvature
+from MD simulations using [MDAnalysis]. We
+aimed to have the following features in the membrane curvature analysis tool:
+
+- Surfaces derived from an `AtomGroup` of reference.
+- Calculation of mean and Gaussian curvature.
+- Implement multiframe and averaged-over-frames analysis.
+- Plug-and-play with visualization libraries to obtain 2D curvature profiles.
+- Data visualization made easy.
+
+<img
+src="https://ojeda-e.github.io/assets/images/patches.png"
```

Comment:
add the image to [public/images](https://github.com/MDAnalysis/MDAnalysis.github.io/tree/master/public/images) (you can also make a subdirectory) please — better to have images locally hosted

Reply:
Then see https://github.com/MDAnalysis/MDAnalysis.github.io#images

Reply:
Given that you have multiple images, make a subdirectory, please!